### PR TITLE
trivia-Avi

### PR DESCRIPTION
**Pull request by Avi-Meshulam**

### CODE REVIEW

File: lesson01.trivia/index.html [hunk 1]
```diff
@@ -1,22 +1,73 @@
...
+            <p id='solution' class='solution'></p>
+        </section>
     </main>
+
+    <script type='text/javascript'>
+        let resultElem = document.getElementById('result');
+        let solutionElem = document.getElementById('solution');
+        document.getElementById('answers').addEventListener('click', e => {
+            if (e.target && e.target.innerText === 'Jupiter') {
+                console.log('Right :)');
```
Comment (yuvalbl) [inline lesson01.trivia/index.html:59]: remove `console.log`

File: lesson01.trivia/index.html [hunk 2]
```diff
@@ -1,22 +1,73 @@
...
+                resultElem.innerText = 'Right!';
+                resultElem.style.color = 'limegreen';
```
Comment (yuvalbl) [inline lesson01.trivia/index.html:61]: Add style using `CSS`

File: lesson01.trivia/index.html [hunk 3]
```diff
@@ -1,22 +1,73 @@
...
+            }
+            else {
+                console.log('Wrong :(');
+                resultElem.innerText = 'Wrong!';
+                resultElem.style.color = 'red';
```
Comment (yuvalbl) [inline lesson01.trivia/index.html:66]: Add style using `CSS` whenever possible (avoid inline style)

File: lesson01.trivia/index.html [hunk 4]
```diff
@@ -1,22 +1,73 @@
...
+            }
+            solutionElem.innerText = 'Jupiter is the largest planet (69,911 km / 43,441 miles) – 1,120% the size of Earth';
```
Comment (yuvalbl) [inline lesson01.trivia/index.html:68]: This is static text - can be in HTML and hidden/displayed using `css`

File: lesson01.trivia/index.html [hunk 5]
```diff
@@ -1,22 +1,73 @@
...
 
+        .question li,
+        .question .result,
+        .question .solution {
+            font-size: 1.5em;
+            line-height: 1.5em;
+        }
+
+        .question li {
+            line-height: 1.5em;
```
Comment (yuvalbl) [inline lesson01.trivia/index.html:25]: can you explain usage of `em` units?

**Review by yuvalbl — APPROVED** [review]
Very nice work and styling!
See comments below

File: lesson01.trivia/index.html (hunk 5 above)
Comment (Avi-Meshulam) [reply lesson01.trivia/index.html:25]: As much as I know, em means relative to the font size of the element, which is 1em by default.

File: lesson01.trivia/index.html (hunk 3 above)
Comment (Avi-Meshulam) [reply lesson01.trivia/index.html:66]: I guess the way to do it is by adding or changing the class list of the element, right?

**Pull request merged**